clients/venice-admin-tool/src/main/java/com/linkedin/venice/AdminTool.java

```diff
@@ -1154,6 +1154,11 @@ static UpdateStoreQueryParams getUpdateStoreQueryParams(CommandLine cmd) {
     booleanParam(cmd, Arg.UNUSED_SCHEMA_DELETION_ENABLED, p -> params.setUnusedSchemaDeletionEnabled(p), argSet);
     booleanParam(cmd, Arg.BLOB_TRANSFER_ENABLED, p -> params.setBlobTransferEnabled(p), argSet);
 
+    longParam(cmd, Arg.MAX_RECORD_SIZE_BYTES, params::setMaxRecordSizeBytes, argSet);
+    if (params.getChunkingEnabled().orElse(false) && params.getMaxRecordSizeBytes().orElse(-1L) == -1L) {
+      params.setMaxRecordSizeBytes(10 * 1024 * 1024);
```

Comment:
@lluwm Do you know a good place I could put the `10 MB` constant?

Reply:
My thinking is that since this is a constant related to a store config, how about defining it in `com.linkedin.venice.utils.StoreUtils`?